.ci/Jenkinsfile

```diff
@@ -29,6 +29,7 @@ pipeline {
     choice(name: 'runTestsSuite', choices: ['all', 'helm', 'ingest-manager', 'metricbeat'], description: 'Choose which test suite to run (default: all)')
     booleanParam(name: "forceSkipGitChecks", defaultValue: false, description: "If it's needed to check for Git changes to filter by modified sources")
     booleanParam(name: "forceSkipPresubmit", defaultValue: false, description: "If it's needed to execute the pre-submit tests: unit and precommit.")
+    string(name: 'SLACK_CHANNEL', defaultValue: 'observablt-robots', description: 'The Slack channel where errors will be posted')
```

Comment:
Would you think a choice param type would be better instead? Having a fixed list of channels will cause not being possible to use a non-valid channel

Reply:
Let's keep this for follow-up discussions